frontend/src/reviews/ReviewRoleDisplay.spec.tsx

```diff
@@ -0,0 +1,85 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import { useGetModelRoles } from 'actions/model'
+import ReviewRoleDisplay from 'src/reviews/ReviewRoleDisplay'
+import {
+  testAccessRequestReview,
+  testAccessRequestReviewNoResponses,
+  testManagerRole,
+  testReleaseReview,
+  testReleaseReviewNoResponses,
+} from 'utils/test/testModels'
+import { describe, expect, vi } from 'vitest'
+
+const mockRoleUtils = vi.hoisted(() => {
+  return {
+    getRoleDisplay: vi.fn(),
+  }
+})
+vi.mock('utils/beta/roles.ts', () => mockRoleUtils)
+
+vi.mock('actions/model', () => ({
+  useGetModelRoles: vi.fn(),
+}))
+
+describe('ReviewRoleDisplay', () => {
+  const testMessageAccess = 'This access needs to be reviewed by the Manager.'
```

Comment:
This highlights that we should update the component so that the message says "access request" instead of just "access". Not for you to do here though :)